consensus/src/quorum_store/proof_manager.rs

```diff
@@ -161,6 +343,9 @@ impl ProofManager {
                             ProofManagerCommand::ReceiveProofs(proofs) => {
                                 self.receive_proofs(proofs.take());
                             },
+                            ProofManagerCommand::ReceiveBatches(batches) => {
```

Comment:
@vusirikala - How do we think about not sending the batches eagerly from the Batch coordinator to the proof manager and instead reading from the DB whenever we need it? This has two advantages

1. We avoid the unnecessary cloning of the batches
2.  There is an attack surface currently that a rouge validator would blow up the memory. We can avoid that by storing it in DB instead.

@bchocho Thoughts?